about.html

```diff
@@ -15,6 +15,7 @@ <h4 id="version"></h4>
         </div>
     </div>
   </body>
+  //here we will write the text file
```

Comment:
What is this comment about?

Reply:
That's for the better understanding of users as we used text/javascript there in the next line